app/components/rdf-input-fields/mandataris-replacement-selector.js

```diff
@@ -144,24 +164,22 @@ export default class MandatarisReplacementSelector extends InputFieldComponent {
   });
 
   @action
-  selectReplacement(mandatarises) {
+  selectReplacement(mandataris) {
     if (this.isDestroyed || this.isDestroying) {
       return;
     }
-    this.replacements = mandatarises;
+    this.replacement = mandataris;
 
     // Retrieve options in store
     const matches = triplesForPath(this.storeOptions, true).values;
 
     // Cleanup old value(s) in the store
     matches
-      .filter((m) => !mandatarises.find((opt) => m.value == opt.uri))
+      .filter((m) => ![mandataris].find((opt) => m.value == opt.uri))
```

Comment:
This feels strange 😄 
Can't you not just do ... or am i not thinking straight 🙈 
```suggestion
      .filter((m) => mandataris.uri !== m.value )

```